src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -384,9 +387,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         InterpolatedString = [Widening] Or (1 << 25)
 
         ' Tuple conversions
+        ''' <summary>
+        ''' Can be combined with <see cref="ConversionKind.Nullable"/> to indicate that the underlying value conversion is a predefined tuple conversion
+        ''' </summary>
         Tuple = (1 << 26)
         WideningTuple = [Widening] Or Tuple
         NarrowingTuple = [Narrowing] Or Tuple
+        WideningNullableTuple = WideningNullable Or Tuple
```

Comment:
Should this be    WideningNullable or Tuple or Value ?     
Nullable is often combined with Value flag - not sure if that is significant here.